internal/image/validate_test.go

```diff
@@ -146,7 +146,7 @@ func TestBuiltinChecks(t *testing.T) {
 			client := ecoci.NewClient(ctx)
 			c.setup(client.(*fake.FakeClient))
 
-			actual, err := ValidateImage(ctx, c.component, p, evaluators, false)
+			actual, err := ValidateImage(ctx, c.component, p, evaluators, false, []app.SnapshotComponent{})
```

Comment:
Could make the test a bit more realistic an include the component that is validated in the array